Header header

uint8 CMD_EMPTY=0
uint8 CMD_VELOCITY=1
uint8 CMD_POS_VEL_ACC=2

uint8 mode

float64[6] q_des
float64[6] qd_des
float64[6] qdd_des
